requirements_production.txt

```diff
@@ -5,9 +5,9 @@ djangorestframework>=3.2.0,<3.4.0
 html5lib>=0.9,<1.0
 jsonfield>=0.9.19,<1.1
 natsort>=3.2,<4.1
-reportlab>=3.0,<3.3
+PyPDF2>=1.25.0,<1.26
+reportlab>=3.0,<3.4
```

Comment:
I tested our pdf with reportlab 3.3.

The reason for the upgrade is, that debian stretch comes with reportlab 3.3. I installed openslides without the need to compile reportlab by installing python3-reportlab. But this was only possible after I changed the supported version to reportlab<3.4.

So in conclusion, to update the version makes the installation a lot easier.